src/main/java/vip/floatationdevice/guilded4j/enums/ExceptionType.java

```diff
@@ -11,13 +11,30 @@
  * The type of {@link vip.floatationdevice.guilded4j.exception.GuildedException} that was thrown.
  */
 @SuppressWarnings("unused")
-public enum ExceptionType
-{
-    /** 400 */ BAD_REQUEST,
-    /** 401 */ UNAUTHORIZED,
-    /** 403 */ FORBIDDEN,
-    /** 404 */ NOT_FOUND,
-    /** 409 */ CONFLICTED,
-    /** 500-504 */ INTERNAL_SERVER_ERROR,
+public enum ExceptionType {
+    /**
+     * 400
+     */
+    BAD_REQUEST,
+    /**
+     * 401
+     */
+    UNAUTHORIZED,
+    /**
+     * 403
+     */
+    FORBIDDEN,
+    /**
+     * 404
+     */
+    NOT_FOUND,
+    /**
+     * 409
+     */
+    CONFLICTED,
+    /**
+     * 500-504
+     */
+    INTERNAL_SERVER_ERROR,
     OTHER
 }
```

Comment:
![barrier](https://user-images.githubusercontent.com/40854260/156746289-06c382c4-b8a3-46e2-97ea-70c05a0e08c8.png)
# nothing approved in this file

Reply:
its neater though